src/index.ts

```diff
@@ -1,4 +1,78 @@
 import { Probot } from "probot";
+import fs from 'fs';
+import { V1Secret } from '@kubernetes/client-node/dist/gen/model/v1Secret';
+const k8s = require('@kubernetes/client-node');
+const { createAppAuth } = require("@octokit/auth-app");
+var cron = require('node-cron');
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
+const k8sContext = kc.getCurrentContext();
+const k8snamespace = (() => {
+  if (k8sContext === 'inClusterContext') {
+    return fs.readFileSync('/var/run/secrets/kubernetes.io/serviceaccount/namespace', 'utf8');
+  } else {
+    return k8sContext.split('/')[0];
+  }
+})();
+console.info(`Operating in the namespace: ${k8snamespace}`);
+
+async function getToken(installId: Number) {
+
+  const appSecret = await k8sApi.readNamespacedSecret("peribolos-as-services", k8snamespace);
+  console.info(`Got the secret peribolos-as-service secrets`);
+  const appID = atob(appSecret.body.data.app_id);
+  const privateKey = atob(appSecret.body.data.private_key);
```

Comment:
Btw, I think we should have the `peribolos-as-a-service` secret rather mounted to the deployment and available in the environment variables or on filesystem, than read/accessed on each per each request. 

Also, I think `atob` is deprecated in NodeJS. You should use something like `Buffer.from(appSecret.body.data.private_key, 'base64').toString('binary')`, however you should not need that at all, since if you mount the secret to the deployment, you have access to the string data directly.

Reply:
That is true, I think i got carried away while trying to run this in the local system :sweat_smile: 
totally forgot that the value is already available as env var.

will refractor accordingly.